packages/eas-cli/src/project/ios/versions.ts

```diff
@@ -0,0 +1,12 @@
+import assert from 'assert';
+
+export function isValidBuildNumber(buildNumber: string): boolean {
+  return !!buildNumber.match(/^\d+(\.\d+)*$/);
+}
+
+export function getNextBuildNumber(buildNumber: string): string {
+  assert(isValidBuildNumber(buildNumber), `Invalid buildNumber ${buildNumber}`);
+  const comps = buildNumber.split('.');
+  comps[comps.length - 1] = String(Number(comps[comps.length - 1]) + 1);
+  return comps.join('.');
+}
```

Comment:
I know this is not exactly the scope of this PR but maybe it's worth using this util in other places - namely, in `src/build/ios/version.ts`?

Reply:
yes, I initially did that, but I had to split code into separate PRs